src/ui/lang.js

```diff
@@ -40,3 +42,13 @@ export function mistakeAlert() {
   }
   alert(messageAlert);
 }
+
+export function showStageID() {
```

Comment:
this works initially, but does not run upon language change (that's why it's not changing). Better to use `langArray`